seed/static/seed/js/services/columns_service.js

```diff
@@ -44,6 +44,16 @@ angular.module('BE.seed.service.columns', []).factory('columns_service', [
       });
     };
 
+    columns_service.delete_column = function (column_id) {
+      return columns_service.delete_column_for_org(user_service.get_organization().id, column_id);
+    };
+
+    columns_service.delete_column_for_org = function (org_id, column_id) {
+      return $http.delete('/api/v3/columns/' + column_id + '/', {
+        params: {organization_id: org_id},
```

Comment:
I added a wrapping object with a params key here since params were not being sent with the request to the backend. 

[That particular backend endpoint does some "smart searching"](https://github.com/SEED-platform/seed/pull/2407/files#diff-f49932cc42b1c65c1fae44800221e8aeR160) for an `organization_id`, getting that ID from a logged in user's "active" (listed in the top right on the UI) organization. This caused the request to happen on the active org vs the org which had it's column settings edited by the user.

Reply:
Thanks for catching this!